### PR TITLE
[IMP][15.0] openupgrade_framework: to not crash, simply not deactivating the views

### PR DESCRIPTION
**Pull request by royle-vietnam**

### CODE REVIEW

**Comment by legalsylvain** [comment]
As said here, I don't think disabling views are the good solution. https://github.com/OCA/OpenUpgrade/pull/3348#issuecomment-1172085534

How do you enable them after ? 

thanks !

**Comment by pedrobaeza** [comment]
This is indeed not deactivating views. Just let the warning.

File: openupgrade_framework/odoo_patch/odoo/addons/base/models/ir_ui_view.py [hunk 1]
```diff
@@ -35,15 +35,7 @@ def _raise_view_error(
                 from_traceback=from_traceback,
             )
         except ValueError:
-            _logger.warning(
```
Comment (pedrobaeza) [inline openupgrade_framework/odoo_patch/odoo/addons/base/models/ir_ui_view.py:38]: Don't remove the warning

**Comment by legalsylvain** [comment]
> This is indeed not deactivating views. Just let the warning.

sorry, I misread.

same remark as pedro. Otherwise, LGTM. Thanks.

File: openupgrade_framework/odoo_patch/odoo/addons/base/models/ir_ui_view.py (hunk 1 above)
Comment (royle-vietnam) [reply openupgrade_framework/odoo_patch/odoo/addons/base/models/ir_ui_view.py:38]: Okey, done

**Review by jcdrubay — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
This can be backported to v14

File: openupgrade_framework/odoo_patch/odoo/addons/base/models/ir_ui_view.py [hunk 2]
```diff
@@ -34,16 +34,16 @@ def _raise_view_error(
...
                 "Can't render custom view %s for model %s. "
-                "Assuming you are migrating between major versions of "
-                "Odoo, this view is now set to inactive. Please "
-                "review the view contents manually after the migration.",
+                "Assuming you are migrating between major versions of Odoo. "
+                "Please review the view contents manually after the migration.\n"
+                "Error: %s",
                 self.xml_id,
                 self.model,
+                e,
```
Comment (pedrobaeza) [inline openupgrade_framework/odoo_patch/odoo/addons/base/models/ir_ui_view.py:45]: One last doubt. Why including `e` in the log? Isn't this too verbose?

Comment (royle-vietnam) [reply openupgrade_framework/odoo_patch/odoo/addons/base/models/ir_ui_view.py:45]: So that we can review faster, otherwise you won't know why it failed

Comment (royle-vietnam) [reply openupgrade_framework/odoo_patch/odoo/addons/base/models/ir_ui_view.py:45]: > This can be backported to v14

Yes, of course

**Review by daiduongnguyen-odoo — APPROVED** [review]

File: openupgrade_framework/odoo_patch/odoo/addons/base/models/ir_ui_view.py (hunk 2 above)
Comment (pedrobaeza) [reply openupgrade_framework/odoo_patch/odoo/addons/base/models/ir_ui_view.py:45]: This is not answering my question. Isn't this too verbose? Can you put an example of the output with the exception included in the log message?

**Comment by pedrobaeza** [comment]
The added text is only one line, so it can be acceptable.

**Pull request merged**